README.md

```diff
@@ -284,6 +284,23 @@ of mangled property names.
 Using the name cache is not necessary if you compress all your files in a
 single call to UglifyJS.
 
+#### Debugging property name mangling
+
+You can also pass `--mangle-debug` in order to mangle property names
+without completely obscuring them. For example the property `o.foo`
+would mangle to `o._$foo$123_` with this option. This allows property mangling
+of a large codebase while still being able to debug the code and identify
+where mangling is breaking things.
+
+Note that by default a random number is added to the name per invocation,
+e.g. the `123` in `_$foo$123_` is random. This is because normal property
+mangling is non-deterministic. It makes sure that if you mangle two scripts
```

Comment:
This sentence is false and should be removed: "This is because normal property mangling is non-deterministic.". 

Uglify prop mangling is deterministic with the same inputs. If you alter your JS inputs or prop name cache, naturally the output prop mangling will change.